test/integration-test.py

```diff
@@ -310,15 +310,72 @@ def run_certificates_per_name_test():
             print("\nCertificates per name test: expected %s not present in output" % s)
             die(ExitStatus.Error)
 
-@atexit.register
-def cleanup():
-    import shutil
-    shutil.rmtree(tempdir)
-    if exit_status == ExitStatus.OK:
-        print("\n\nSUCCESS")
-    else:
-        if exit_status:
-            print("\n\nFAILURE %d" % exit_status)
+default_config_dir = os.environ.get('BOULDER_CONFIG_DIR', '')
+if default_config_dir == '':
+    default_config_dir = 'test/config'
+
+def run_admin_revoker_test():
+    email_addr = "js.integration.test@letsencrypt.org"
+    cert_file = os.path.join(tempdir, "ar-cert.der")
+    cert_file_pem = os.path.join(tempdir, "ar-cert.pem")
+    key_file = os.path.join(tempdir, "key.pem")
+    # Issue certificate for serial-revoke test
+    if subprocess.Popen('''
+        node test.js --email %s --domains ar-test.com \
+          --certKey %s --cert %s --challType http-01 && \
+        openssl x509 -in %s -out %s -inform der -outform pem
+        ''' % (email_addr, key_file, cert_file, cert_file, cert_file_pem),
+        shell=True, cwd=JS_DIR).wait() != 0:
+        print("\nIssuing failed")
+        die(ExitStatus.NodeFailure)
+    # Extract serial from certificate
+    try:
+        serial = subprocess.check_output("openssl x509 -in %s -noout -serial | cut -c 8-" % (cert_file_pem),
+                                         shell=True, stderr=subprocess.STDOUT)
+    except subprocess.CalledProcessError as e:
+        print "Failed to extract serial: %s" % (e.output)
+        die(ExitStatus.PythonFailure)
+    serial = serial.rstrip()
+    # Revoke certificate by serial
+    config = default_config_dir + "/admin-revoker.json"
+    print("./bin/admin-revoker serial-revoke --config %s %s %d" % (config, serial, 1))
+    if subprocess.Popen("./bin/admin-revoker serial-revoke --config %s %s %d" % (config, serial, 1),
+                        shell=True).wait() != 0:
+        print("Failed to revoke certificate")
+        die(ExitStatus.RevokerFailure)
+    # Wait for OCSP response to indicate revocation took place
+    ee_ocsp_url = "http://localhost:4002"
+    wait_for_ocsp_revoked(cert_file_pem, "test/test-ca2.pem", ee_ocsp_url)
+
+    # Issue certificate for auth-revoke test
+    try:
+        output = subprocess.check_output("node test.js --email ar@test.com --domains ar-auth-test.com --abort-step startChallenge",
+                            shell=True, cwd=JS_DIR)
+    except subprocess.CalledProcessError as e:
+        print "Failed to create authorization: %s" % (e.output)
+        die(ExitStatus.NodeFailure)
+    # Get authorization URL from last line of output
+    lines = output.rstrip().split("\n")
+    if not lines[len(lines)-1].startswith("authorization-url="):
```

Comment:
You can replace the usage of `lines[len(lines)-1]` here and on L362 with `lines[-1]`.

Can you put the "authorization-url=" into a var (e.g. `prefix`) and then do `if not lines[-1].startswith(prefix):` here, and on L362: `url = lines[-1][len(prefix):]` - the `18:` is a little bit magic.